README.en.md

```
@@ -0,0 +1,114 @@
+# Files Converter
+
+## Overview
+
+**files-converter** - This is a tool that converts source configuration files, or a *.cf file, to a *.cfg file, and vice versa.
```

Markdownlint

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)